src/plugins/objects/livecounter.ts

```
@@ -22,66 +22,66 @@ export class LiveCounter extends LiveObject<LiveCounterData, LiveCounterUpdate>
+    obj._mergeInitialDataFromCreateOperation(objectOperation);
     return obj;
   }
 
   /**
    * @internal
    */
-  static createCounterIncMessage(objects: Objects, objectId: string, amount: number): StateMessage {
+  static createCounterIncMessage(objects: Objects, objectId: string, amount: number): ObjectMessage {
     const client = objects.getClient();
 
     if (typeof amount !== 'number' || !isFinite(amount)) {
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)